### DISCUSSION TITLE
Dashboard save configuration

### DISCUSSION THREAD

**Discussion by t347s** [alexjustesen/speedtest-tracker#261] [Ideas]
Hi,
A have a design and functionality proposal for you, the time selection (Last 24h, Last week, Last month) to be sticky per user, or somekind of configuration option, also a bulk option should be nice, it's quite annoying to select an option for each widget (Download / Upload (mbps) ,  Ping (ms),  Jitter (ms) ).

![image](https://user-images.githubusercontent.com/31503790/209098109-f62abd4d-bf9b-4d4c-80c1-6c26cccfeaed.png)

Thank you!

**Comment by alexjustesen** [comment]
I agree this is something that should be added, got a couple of items on my list before circling back around to improving charts

**Comment by Aschefr** [comment]
Hi ! I came to put in the same request : I would love that the graph always show Week or Month instead of 24H as I run the speedtest once a day. 24h show only one point without context. Thanks a lot for this docker anyway !
